src/fhirdefs/FHIRDefinitions.ts

```diff
@@ -103,6 +107,43 @@ export class FHIRDefinitions implements Fishable {
     return this.predefinedResources.get(file);
   }
 
+  resetPredefinedResources() {
+    this.predefinedResources = new Map();
+  }
+
+  fishForPredefinedResource(item: string, ...types: Type[]): any | undefined {
+    const resource = this.fishForFHIR(item, ...types);
+    if (
+      resource &&
+      this.allPredefinedResources().find(
+        predefResource =>
+          predefResource.id === resource.id && predefResource.resourceType === resource.resourceType
```

Comment:
Why test on `id` and `resourceType` rather than `url`?  I think `url` is really the only _globally unique_ identifier.  It's technically possible (and I think valid) to have two different resources w/ the same `id` and `resourceType` (as long as they have different URLs).

Reply:
Also, you're using `url` as the matcher in `Processing.ts` so it seems like it would be more consistent.

Reply:
Right, I agree they should match, so I need to fix that either way, but I went with `resourceType` and `id` because I don't think matching on `url` works for Instances right? Because an Instance may not have a `url`. But we do require that an Instance have a `resourceType` and `id`, since those are used to generate the file names.

Reply:
Oh...  I wasn't thinking of instances...  Yeah, I guess you're right.  Can we match on all three then?  In the case of instances, it should still match since `url` would be `undefined` for both.  And in my edge case it would properly detect things that have matching `id` and `resourceType` but different `url`.